concourse/pipelines/create-microbosh.yml

```diff
@@ -225,7 +240,8 @@ jobs:
 
             ./build_manifest.sh \
               ../../../bosh-secrets/bosh-secrets.yml \
-              ../../../terraform-outputs/terraform-outputs.yml \
+              ../../../terraform-outputs/bosh.terraform-outputs.yml \
```

Comment:
Can you refactor it in a way that is done [here](https://github.com/alphagov/paas-cf/pull/28/files#diff-35adbaaa48f824ce18c1838d3dc9342fR208)

You can specify `MANIFESTS_DIR` as a task parameter that acts a a prefix dir for manifests.